src/promptflow/promptflow/_sdk/_serving/monitor/data_collector.py

```diff
@@ -49,3 +49,22 @@ def collect_flow_data(self, input: dict, output: dict, req_id: str = None, clien
             self.logger.warn(f"Load mdc related module failed: {e}")
         except Exception as e:
             self.logger.warn(f"Collect flow data failed: {e}")
+
+    def collect_feedback_data(self, rating: str, message: str, conversation: list, req_id: str = None, client_req_id: str = None):
```

Comment:
feedback collection can use its own collector which means the collected feedback data can be stored in its own dataset instead of in output dataset.

For Long term, it would be better to also use OTEL to collect the data, benefit of this is that it can work without MDC